examples/Autopoint/README.md

```diff
@@ -0,0 +1,58 @@
+
+## Installation of the required libraries
+C++ 17 is required
```

Comment:
where is this come from? autodiff?

Reply:
https://github.com/autodiff/autodiff/ Autodiff is a C++ 17 library and it uses a lot of modern features